website/content/docs/enterprise/federation.mdx

```diff
@@ -1,10 +1,8 @@
 ---
 layout: docs
-page_title: Consul Enterprise Advanced Federation
+page_title: Advanced Federation (Enterprise)
```

Comment:
If we're focusing on including the feature name, maybe something like "Network Areas for WAN Federation" or "Federation with Network Areas".

I like the simplicity of "Federated Network Areas", but you aren't federating the network areas, you are federating the datacenters and using network areas to simplify that a bit.

Cluster peering will be strictly better than network areas for network topology flexibility. The only reason you'd use network areas instead is if you want to continue to use WAN Federation rather than cluster peering for your multi-DC setup. Network areas are the most flexible you can get within WAN federation.